frontend/src/features/ratings/RatingsFilter.tsx

```diff
@@ -22,14 +24,20 @@ function RatingsFilter() {
         {t('options')}
       </CollapseButton>
       <Collapse in={expanded} timeout="auto" unmountOnExit>
-        <Grid container spacing={4} sx={{ marginBottom: '8px' }}>
-          <Grid item xs={12} sm={6} md={5}>
+        <Grid container spacing={2} mb={1} justifyContent="space-between">
+          <Grid item xs={12} sm={6} md={3}>
             <IsPublicFilter
               value={filterParams.get('isPublic') ?? ''}
               onChange={(value) => setFilter('isPublic', value)}
             />
           </Grid>
-          <Grid item xs={12} sm={6} md={5}>
+          <Grid item xs={12} sm={6} md={5} lg={4}>
+            <RatingOrderByInput
+              value={filterParams.get('orderBy') ?? ''}
+              onChange={(value) => setFilter('orderBy', value)}
+            />
```

Comment:
2 small things about the behavior of this input:
(They are not blocking, and may be addressed separately if considered relevant.)

1. By default when accessing the /ratings page, no value for `orderBy` is selected. The order will depend on the default order implemented in the backend. I think it would be a slightly better UX to have a selected value at all time, controlled by the frontend. It may require to extend the implementation of `useListFilter()` to pass a default value when none is present in the URL.

2. The order of items with `?orderBy=-last_compared_at` may be different from the default order (when no explicit orderBy is defined). That's because the backend implementation uses the primary key to break ties by default:
https://github.com/tournesol-app/tournesol/blob/6eda6020a9cb56711c23631f7e82efc4606327de/backend/tournesol/views/ratings.py#L28
I think it would make sense to append `"-pk"` in the backend implementation when ordering entities, to keep a deterministic order in all cases.

Reply:
Thank you, I'll try to fix both points in this PR.

Regarding the point 1. I've been able to add a default filter value by updating the `RatingsFilter`. Do you think it's better to extend `useListFilter()` instead? I currently don't know if the hook `useListFilter()` is a good place to handle the default values, mainly because my lack of knowledge about React's hooks.

See: https://github.com/tournesol-app/tournesol/pull/1181/commits/b4757890b3927f35022ba37bb8b5715fd6907152

This commit also includes the point 2.

Reply:
I finally fixed the point 1 as suggested.

https://github.com/tournesol-app/tournesol/pull/1181/commits/c93faf66ec808dfa0e3e0135bfe74d042c149fd9